tests/modules/hooks.c

```diff
@@ -290,13 +290,12 @@ void keyInfoCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub, void
     REDISMODULE_NOT_USED(e);
 
     RedisModuleKeyInfoV1 *ei = data;
-    RedisModuleString *key = ei->key;
-    const char *keyname = RedisModule_StringPtrLen(key, NULL);
+    RedisModuleKey *kp = ei->key;
+    const char *keyname = RedisModule_StringPtrLen(RedisModule_GetKeyNameFromModuleKey(kp), NULL);
     RedisModuleString *event_keyname = RedisModule_CreateStringPrintf(ctx, "key-info-%s", keyname);
     LogStringEvent(ctx, RedisModule_StringPtrLen(event_keyname, NULL), keyname);
     RedisModule_FreeString(ctx, event_keyname);
 
-    RedisModuleKey *kp = RedisModule_OpenKey(ctx, key, REDISMODULE_READ);
```

Comment:
maybe we should still keep some test doing RM_OpenKey, or RM_Call so that we can demonstrate that the key was not yet removed from the database and that doing dictFind won't mess things up..
i.e. we went through so much in order to support it, let's keep it covered by a test (in addition to using the key that's provided by the callback)